python/pyspark/sql/column.py

```diff
@@ -124,6 +124,35 @@ def _(self, other):
     return _
 
 
+like_doc = """ Return a Boolean :class:`Column` based on a SQL LIKE match.\n
+               :param other: a SQL LIKE pattern\n
+               See :func:`pyspark.sql.Column.rlike` for a regex version\n
+
+               >>> df.filter( df.name.like('Al%') ).collect()
+               [Row(name=u'Alice', age=1)]
+            """
+rlike_doc = """ Return a Boolean :class:`Column` based on a regex match.\n
+                :param other: an extended regex expression\n
+
+                >>> df.filter( df.name.rlike('ice$') ).collect()
+                [Row(name=u'Alice', age=1)]
+            """
+endswith_doc = ''' Return a Boolean :class:`Column` based on matching end of string.\n
+                   :param other: string at end of line (do not use a regex `$`)\n
+                   >>> df.filter(df.name.endswith('ice')).collect()
+                   [Row(name=u'Alice', age=1)]
+                   >>> df.filter(df.name.endswith('ice$')).collect()
+                   []
+               '''
+startswith_doc = ''' Return a Boolean :class:`Column` based on a string match.\n
```

Comment:
Ah, thank you! I couldn't figure out how to get the docs from not showing up as separate functions, but this solved it. I added `_` to the front of the docstrings, and moved them into the `Column` class, immediately before the `_bin_op` definitions.